src/components/Card/src/CardHeader/card-header.css

```diff
@@ -0,0 +1,7 @@
+.denhaag-card__header {
+    --denhaag-card__header-font-style: normal;
+    --denhaag-card__header-font-weight: bold;
+    --denhaag-card__header-font-size: 16px;
+    
+    --denhaag-card__header-line-height: 24px;
```

Comment:
`__` in a custom property should probably be `-`.

Reply:
Done